include/boost/numeric/ublas/matrix_expression.hpp

```diff
@@ -88,6 +88,23 @@ namespace boost { namespace numeric { namespace ublas {
         }
 #endif
 
+#ifndef BOOST_UBLAS_REFERENCE_CONST_MEMBER
```

Comment:
You added a number of new accessors to existing types. Do you have tests for them ?

Reply:
Yes. I have tests in test/tensor. Yet there are no unit-tests in ublas testing purely expression templates.

Reply:
I will mark this as resolved now. There is no simple solution to unit-testing matrix expression templates.